<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2560</width>
    <height>1440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>2560</width>
    <height>1440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="baseLayout">
      <item>
       <layout class="QHBoxLayout" name="topLineLayout">
        <item>
         <widget class="QPushButton" name="pushButton_run">
          <property name="minimumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/resources/run_button.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_stop">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/resources/stop_button.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_single">
          <property name="minimumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/resources/single_button.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">7image: url(:/resources/7-200Z9142449E4.jpg);
color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="contentLayout">
        <item>
         <layout class="QVBoxLayout" name="leftLayout">
          <item>
           <widget class="QWidget" name="widget_2" native="true"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_screenshot">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/resources/screenshot.jpg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_savewave">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/resources/savewave.jpg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="middleLayout">
          <item>
           <widget class="XYView" name="widget_xyview" native="true"/>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>90</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-color: rgb(6, 6, 6);
font: 9pt &quot;Ubuntu&quot;;
font-color:rgb(33,44,55)</string>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Source Code Pro'; font-size:18pt; color:#ffaa00;&quot;&gt;横坐标：频率(GHz)	纵坐标：强度(dBm)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rightLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2560</width>
     <height>48</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_load"/>
    <addaction name="action_select_save"/>
    <addaction name="action_save_as"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="action_preference"/>
    <addaction name="action_calibration"/>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>控件</string>
    </property>
    <addaction name="action_EDFA"/>
    <addaction name="action_laser"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_load">
   <property name="text">
    <string>导入文件</string>
   </property>
  </action>
  <action name="action_select_save">
   <property name="text">
    <string>选择保存路径</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>另存为
</string>
   </property>
  </action>
  <action name="action_EDFA">
   <property name="text">
    <string>EDFA控制器</string>
   </property>
  </action>
  <action name="action_laser">
   <property name="text">
    <string>激光器控制器</string>
   </property>
  </action>
  <action name="action_preference">
   <property name="text">
    <string>偏好</string>
   </property>
  </action>
  <action name="action_calibration">
   <property name="text">
    <string>校准</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XYView</class>
   <extends>QWidget</extends>
   <header location="global">Axis/XYView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
